swan-lake/resources/featured-scenarios/build-a-data-service-in-ballerina.md

```diff
@@ -59,224 +55,229 @@ CREATE TABLE Company.Employees (
-    int employee_id?;
+type Employee record {|
     string first_name;
     string last_name;
     string email;
     string phone;
-    time:Date hire_date;
-    int? manager_id;
     string job_title;
+    int employee_id?;
+    int? manager_id;
+    time:Date hire_date;
 |};
 ```
 
-### Add the MySQL driver
-
-The MySQL driver JAR is necessary to connect to and interact with a MySQL server. Select one out of the methods below to add it.
-
-1. Import the `ballerinax/mysql.driver` package in your `main.bal` file. This package bundles the latest MySQL driver so that the MySQL connector can be used in Ballerina packages easily.
-
-   ```ballerina
-   import ballerinax/mysql.driver as _;
-   ```
-   
-2. Update the `Ballerina.toml` file with the Maven dependency params for the MySQL driver.
-   ```toml
-   [[platform.java11.dependency]]
-   groupId = "mysql"
-   artifactId = "mysql-connector-java"
-   version = "8.0.26"
-   ```
-   
-3. Download the driver JAR manually and update the path in the `Ballerina.toml` file
-   ```toml
-   [[platform.java11.dependency]]
-   path = "/path/to/mysql/driver.jar”
-   ```
-   
 ### Define MySQL configurations
 
-In the package directory, create a new file named `Config.toml` and specify the configurations below to connect to the MySQL database.
+Follow the steps below to define the MySQL configurations.
 
-```toml
-USER="root"
-PASSWORD="rootPassword"
-HOST="localhost"
-PORT=3306
-DATABASE="Company"
-```
+1. In the package directory, create a new file named `Config.toml` and specify the configurations below to connect to the MySQL database.
 
-To redefine the above variables and access them from within your Ballerina program, add the code below in the `main.bal` file.
+    ```toml
+    USER="root"
+    PASSWORD="rootPassword"
+    HOST="localhost"
+    PORT=3306
+    DATABASE="Company"
+    ```
 
-```ballerina
-configurable string USER = ?;
-configurable string PASSWORD = ?;
-configurable string HOST = ?;
-configurable int PORT = ?;
-configurable string DATABASE = ?;
-```
+2. Define configurable variables to redefine the above variables and access them from within your Ballerina program.
+
+    >**Info:** For more information on defining configurable variables in Ballerina, see [Provide values to configurable variables](/learn/provide-values-to-configurable-variables/).
+
+    ![Create configurable variable](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/create-configurable-variable.gif)
 
->**Note:** For more information on defining configurable variables in Ballerina, see [Provide values to configurable variables](/learn/provide-values-to-configurable-variables/).
+    The generated configurable variables will be as follows.
+
+    ```ballerina
+    configurable string USER = "root";
+    configurable string PASSWORD = "rootPassword";
+    configurable string HOST = "localhost";
+    configurable int PORT = 3306;
+    configurable string DATABASE = "Company";
```

Comment:
Better to use normal variable naming conventions without upper caps for configurable. @MaryamZi wdyt?

Reply:
Yeah. We use all uppercase only for constants.

Reply:
Updated.